act-rules-format.bs

```diff
@@ -66,7 +66,9 @@ Accessibility Requirements {#structure-accessibility-requirements}
 
 An ACT Rule MUST identify the accessibility requirements to which the rule maps. For example, WCAG 2.0 Success Criterion 1.1.1. An ACT Rule is a complete or partial test for one or more accessibility requirements.
 
-Outcomes from an ACT Rule MUST be consistent with the accessibility requirement, e.g. a rule only returns the outcome Fail when the content fails the accessibility requirement. This means that the rule maps to the accessibility requirement, as opposed to it merely being related to the requirement, thematically or otherwise.
+Outcomes from an ACT Rule that is not part of an [aggregation group](#grouping) MUST be consistent with the accessibility requirement, e.g. a rule only returns the outcome Fail when the content fails the accessibility requirement. This means that the rule maps to the accessibility requirement, as opposed to it merely being related to the requirement, thematically or otherwise.
```

Comment:
"rule only returns the outcome Fail when the content fails the accessibility requirement" That's not correct. Rules can return fail, even if the requirement is passed. If only one rule in a group has to pass for the AR to pass, than any number of rules can fail without the AR failing.

Reply:
This section is about "ACT Rule that is not part of an [aggregation group](#grouping)". Next section is about "ACT Rules that are part of aggregation groups".